local_pathfinding/objectives.py

```diff
@@ -111,14 +110,17 @@ class MinimumTurningObjective(Objective):
     Attributes:
         goal_x (float): The x coordinate of the goal state
         goal_y (float): The y coordinate of the goal state
-        heading_degrees (float): The heading of the sailbot in degrees
+        heading (float): The heading of the sailbot in radians
```

Comment:
Range is [0, 2pi)

https://github.com/UBCSailbot/custom_interfaces/blob/main/msg/HelperHeading.msg